src/app/containers/Image/index.jsx

```diff
@@ -22,6 +23,21 @@ const getCopyright = copyrightHolder => {
   return copyrightHolder;
 };
 
+const getSrcSet = imgUrl => {
```

Comment:
Apparently multiline suggestions aren't a thing yet? 😔 

I think this function would a lot clearer as:
```
const getSrcSet = imgUrl => {
  const srcSetUrls = srcSetValues.map(value =>
    imgUrl.replace(DEFAULT_IMAGE_RES, value),
  );

  return srcSetUrls.join(', ');
};
```